include/flamegpu/visualiser/ModelVis.h

```diff
@@ -160,7 +161,7 @@ class ModelVis {
      * @param modelPath Path of the model on disk
      * @param texturePath Optional path to a texture fore the model on disk
      */
-    StaticModelVis addStaticModel(const std::string &modelPath, const std::string &texturePath = "");
+    StaticModelVis newStaticModel(const std::string &modelPath, const std::string &texturePath = "");
```

Comment:
This needs to be listed as a braking change in the changelog for the next release. Probably better to do it now rather than hope it gets picked up when manually making the next changelog (which will be enourmous).

Also needs updating in the docs PR, but I've changed my review on that to reflect this already.

The change itself is a good one for consistency though, and now is the time to make breaking changes.

Reply:
![image](https://user-images.githubusercontent.com/742154/195608132-7b4be01a-78f2-4900-8985-b14660a03cf4.png)
Already was marked as breaking, will update docs.